doc/source/parameters/unresolved_cfd-dem/void_fraction.rst

```diff
@@ -5,23 +5,23 @@ In this subsection, all parameters required for the calculation of the void frac
 
 .. code-block:: text
 
-subsection void fraction
-   set mode = dem
-   set read dem = true
-   set dem file name = dem
-   set l2 smoothing factor = 0
-   set l2 lower bound = 0
-   set l2 upper bound = 1
-   set bound void fraction = false
-end
+   subsection void fraction
+      set mode = dem
+      set read dem = true
+      set dem file name = dem
+      set l2 smoothing factor = 0
+      set l2 lower bound = 0
+      set l2 upper bound = 1
+      set bound void fraction = false
+   end
 
 * The "mode" parameter allows the user to choose the method for void fraction calculation. Currently, there are two methods implemented. The first one is to calculate the void fraction from `function`. In this case, an additional subsection is required to insert the function such as:
 
 .. code-block:: text
 
-     subsection function
-     set Function expression = 0.5 + 0.25*sin(pi*x)*sin(pi*y)
-     end
+   subsection function
+   set Function expression = 0.5 + 0.25*sin(pi*x)*sin(pi*y)
+   end
      
 If the ``mode`` chosen is ``dem``, then the void fraction is calculated using the Particle Centered Method. In this method, the remaining parameters are required.
 * The ``read dem`` allows us to read an already existing dem simulation result which can be obtained from checkpointing the Lethe-DEM simulation. This is important as the gls_vans solver requires reading an initial dem triangulation and particle information to simulate flows in the presence of particles. 
```

Comment:
There is something weird here. I think the bullet points are not working correctly? I'm not sure why?

Reply:
let me check it out

Reply:
Just fixed it in last commit ;).

Reply:
I think a space between the sentence that ends with "required." and the one that starts with "* The ``read dem``" is missing.